app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -11,12 +11,14 @@
 .gem-c-pagination__item {
   @include govuk-font($size: 16, $line-height: (20 / 16));
   list-style: none;
+  border-top: 1px solid $govuk-border-colour;
+  margin-bottom: 16px;
 }
 
 .gem-c-pagination__link {
   @extend %govuk-link;
   display: block;
-  padding: govuk-spacing(3);
+  padding: govuk-spacing(3) govuk-spacing(0) govuk-spacing(4) govuk-spacing(0);
```

Comment:
I don't think there's any point putting `govuk-spacing(0)` into our sass, it's more readable to just put `0`.

Reply:
have removed and replaced with 0 as suggested